ci-operator/step-registry/ipi/conf/vsphere/folder/ipi-conf-vsphere-folder-commands.sh

```diff
@@ -15,18 +20,20 @@ source "${SHARED_DIR}/govc.sh"
 unset SSL_CERT_FILE
 unset GOVC_TLS_CA_CERTS
 
-FOLDER="/$vsphere_datacenter/vm/$vsphere_datacenter"
-SUB_FOLDER="$FOLDER/${NAMESPACE}-${UNIQUE_HASH}"
-
-if govc folder.info "$SUB_FOLDER"; then
-  echo "$SUB_FOLDER already exist, no need to create"
-else
-  if govc folder.info "$FOLDER"; then
-    echo "$FOLDER already exist, no need to create"
+if [ "${FOLDER}" == "default" ]; then
```

Comment:
@jinyunma I remember there is some pre-created resource to use in some vsphere step, but I can not remember the resource is folder or some other things, can you help me recall it ?

Reply:
Do you mean custom resourcepool? In Prow CI, cluster is always created in custom resorucepool, which is configured in https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh#L125, and this resourcepool is pre-created on vsphere, and used directly in prow step.